e2e/test/scenarios/permissions/impersonated.cy.spec.js

```diff
@@ -272,8 +273,7 @@ describeEE("impersonated permission", () => {
       cy.findByRole("dialog").findByText("Edit settings").click();
 
       // Page leave confirmation should be on top
```

Comment:
Another test that wasn't working because of the focus trap. Just a one-liner change but it accomplishes the same goal and makes sure that the confirmation popup is still above the impersonation modal